en_us/troubleshooting.md

```diff
@@ -6,6 +6,55 @@ For the most up-to-date version of this troubleshooting information, [check the
 GitHub
 repository](https://github.com/puppetlabs/puppet-quest-guide/blob/master/en_us/troubleshooting.md).
 
+### I can't find the Quest Guide for the Learning VM
+
+The Quest Guide is hosted by the VM itself, at http://<IPADDRESS>, where is the
```

Comment:
I am not sure if there is a word missing after "where" or if it is just the remnant of an edited sentence. But I think it needs to be adjusted.